lib/modules/easterEgg.js

```diff
@@ -23,9 +25,27 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
-	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
+	}
+
+	konami.addEvent(document, "keydown", function (e, ref_obj) {
```

Comment:
That's what it was feeling like. If I get both of your blessings then I shall proceed with this.